drivers/focuser/moonlite.cpp

```diff
@@ -616,7 +614,16 @@ bool MoonLite::sendCommand(const char * cmd, char * res, bool silent)
     if (res == nullptr)
         return true;
 
-    if ((rc = tty_nread_section(PortFD, res, ML_RES, ML_DEL, ML_TIMEOUT, &nbytes_read)) != TTY_OK)
```

Comment:
So you are considering the GV command returns a fixed-length block of data of size 2, then the remainder is flushed. OK for me as far as I know about the Moonlite protocol. Rest of the file is OK too.